activiti-cloud-events-adapter/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Default values for Maven projects.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+
+alfresco-infrastructure:
```

Comment:
@cristina-sirbu same here.. this chart is not aligned with all the other charts.. so we will have issues if we include this.. I know that this is not your fault.. but we need to figure out how to move forward to be able to merge this.

Reply:
@Salaboy, not sure what you mean.

We're essentially using `alfresco-infrastructure` to bring in ActiveMQ, everything else is disabled.

Reply:
@rgauss that means that to keep it simple we can just add the dependency to the ActiveMQ chart instead of adding a chart that has tons of things and disabling everything.. We definitely need to align on that front but to move this forward I would like to keep it simple and just ActiveMQ which is the only piece needed to run the events-adapter.